### PR TITLE
Lightning Talk Polling - Chengjie ZHANG

### PR DESCRIPTION
**Pull request by JoyCJay**
http://42.192.95.242/

Open 2 tabs to test. See README.md for details

### CODE REVIEW

File: back-end/dist/DB/memoryDB.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"use strict";
```
Comment (rankun203) [inline back-end/dist/DB/memoryDB.js:1]: Why would you commit `dist` folder?

Comment (JoyCJay) [reply back-end/dist/DB/memoryDB.js:1]: For the sake of easy deployment, l have some modification in index.html, but this increment was not taken into account due to .gitignore. l know it's not a good practice, but it's a fast and easy way.

There are many others solutions like do more config in webpack (permanently)
 or temporally transfer the files by ssh etc...

Comment (rankun203) [reply back-end/dist/DB/memoryDB.js:1]: What about build it into a docker image?

Comment (JoyCJay) [reply back-end/dist/DB/memoryDB.js:1]: Yes, l does another good way. Push it to a public docker hub repo and pull,run it directly on the host server.

**Comment by rankun203** [comment]
I think we have gathered enough information with the PR, thanks 👍

**Pull request closed without merging**